### ISSUE TITLE
Can't view submitted solution in Python track

### ISSUE THREAD

**Issue by deepu9** [exercism/exercism#4817]
Hi,
I've started Python track and submitted my Hello World solution, the CLI says successfully submitted and displays solution link, but when I click on it, see attached

![Screen Shot 2019-04-16 at 11 22 26 pm](https://user-images.githubusercontent.com/3803259/56212989-855bdc80-609e-11e9-9d08-93988d64025b.png)

I've tried submitting multiple times, but no luck. See below troubleshoot output

![Screen Shot 2019-04-16 at 11 23 54 pm](https://user-images.githubusercontent.com/3803259/56213164-dd92de80-609e-11e9-9e32-0e0e2f12cc7f.png)

API key is set. I just hide it.

Can anyone help me out. Thanks

**Comment by iHiD** [comment]
My guess would be that your API key is for a different user to the one you're logged in as. Maybe you have two accounts?

**Comment by deepu9** [comment]
I have only one github account. May be I'll generate new one or check the existing one. Thanks for the reply @iHiD

**Comment by iHiD** [comment]
Have you also signed up with your email address? Do you use a shared computer? You can check the token that the cli is using and cross reference against the token on the website to be sure.

**Comment by deepu9** [comment]
Nope, I've signed into Exercism with my github account. I just refreshed my token and set that in my CLI, but no luck

**Comment by deepu9** [comment]
https://exercism.io/my/solutions/287ab2c0d50d4b13bdf39806017653c4

Account name: deepu9

![Screen Shot 2019-04-17 at 7 50 02 pm](https://user-images.githubusercontent.com/3803259/56279099-d5907880-614a-11e9-908e-bade6093ec6b.png)

![Screen Shot 2019-04-17 at 7 50 25 pm](https://user-images.githubusercontent.com/3803259/56279114-dde8b380-614a-11e9-8c98-dd2bef9e98ee.png)

**Comment by NobbZ** [comment]
> deepu9

Is this a typo? When I follow the link, it tells me "deepu09" were the owner of the submission in the pages title.

**Comment by deepu9** [comment]
Ya, I can see the same. Sorry I missed it, but don't recognise that account. Let me check

**Comment by deepu9** [comment]
Hi @NobbZ, I found another account and I can see python solution going into it. Thanks

The only thing I don't understand is I've changed the token in my CLI config, but it still submit to deepu09 account instead of deepu9. Do you know why?

**Comment by deepu9** [comment]
I found the issue. It's `metadata.json`. When I was still using deepu09 token, `metadata.json` file stores url and id of solution submitted to deepu09. So even though I changed the token after that, the `metadata.json` under `.exercism` folder in the exercise hasn't updated.

I've deleted the `.exercism` folder and downloaded the exercise again. Now when I submit, it goes to deepu9 account.

Thanks to all who helped me fix this issue.
